### PR TITLE
WIP NOMRG Benchmark configurations

### PR DESCRIPTION
**Pull request by mathurinm**

### CODE REVIEW

**Comment by tomMoral** [comment]
:fire:

File: datasets/meg.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    from mne.datasets import sample
+    from mne.inverse_sparse.mxne_inverse import _prepare_gain
+    from mne.utils import logger
+    logger.setLevel(0)
+
+
+class Dataset(BaseDataset):
+    name = "MEG"
+    install_cmd = "conda"
+    requirements = ["mne"]
```
[the remark this answers is not in the document]
Comment (mathurinm) [reply datasets/meg.py:17]: May be we can even go with pip:mne  which takes 1 s ? @agramfort wdyt ?

File: datasets/meg.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+from benchopt import BaseDataset
+from benchopt import safe_import_context
+
+
+with safe_import_context() as ctx:
```
Comment (tanglef) [inline datasets/meg.py:5]:
```suggestion
with safe_import_context() as import_ctx:
```

**Review by tanglef — COMMENTED** [review]
`import_ctx` is the reason why `benchopt info` wasn't right

File: datasets/meg.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (agramfort) [reply datasets/meg.py:17]: yes use pip it's enough here.

File: datasets/meg.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    from mne.datasets import sample
+    from mne.inverse_sparse.mxne_inverse import _prepare_gain
+    from mne.utils import logger
+    logger.setLevel(0)
+
+
+class Dataset(BaseDataset):
+    name = "MEG"
+    install_cmd = "conda"
+    requirements = ["mne"]
```
Comment (mathurinm) [inline datasets/meg.py:17]:
```suggestion
    requirements = ["pip:mne"]
```

**Comment by mathurinm** [comment]
Changes to merge have been addressed in others PRs

**Pull request closed without merging**